ObsStudioPlugin/src/ObsPlugin/ObsStudioPlugin.csproj

```diff
@@ -1,268 +1,153 @@
+    <EmbeddedResource Include="Icons\STREAM_DisableStudioMode2.png" />
+    <EmbeddedResource Include="Icons\STREAM_EnableStudioMode.png" />
+    <EmbeddedResource Include="Icons\STREAM_SaveReplay.png" />
+    <EmbeddedResource Include="Icons\STREAM_StartReplayBuffer.png" />
+    <EmbeddedResource Include="Icons\STREAM_StartStreamingGreen.png" />
+    <EmbeddedResource Include="Icons\STREAM_StartStreamingRed.png" />
+    <EmbeddedResource Include="Icons\STREAM_StopReplayBuffer.png" />
+    <EmbeddedResource Include="Icons\STREAM_ToggleRecord1.png" />
+    <EmbeddedResource Include="Icons\STREAM_ToggleRecord2.png" />
+    <EmbeddedResource Include="Icons\VirtualWebcam.png" />
+    <EmbeddedResource Include="Icons\VirtualWebcamOff.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Icons\Workspaces.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Icons\STREAM_Transition.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Icons\STREAM_RecordPause.png" />
+    <EmbeddedResource Include="Icons\STREAM_RecordResume.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="metadata\Icon256x256.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Icons\FilterOff.png" />
+    <EmbeddedResource Include="Icons\FilterOn.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Icons\AudioDisabled.png" />
+    <EmbeddedResource Include="Icons\FilterDisabled.png" />
+    <EmbeddedResource Include="Icons\SceneDisabled.png" />
+    <EmbeddedResource Include="Icons\SourceDisabled.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
+    <PackageReference Include="obs-websocket-dotnet" Version="5.0.0.3" />
+    <PackageReference Include="System.Data.DataSetExtensions" Version="4.5.0" />
+    <PackageReference Include="System.Reactive" Version="4.3.2" />
+    <PackageReference Include="Websocket.Client" Version="4.4.43" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="6.0.0" />
+    <PackageReference Include="System.Threading.Channels" Version="8.0.0" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="Support\Support.TEMPLATE.cs" />
+  </ItemGroup>
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
+    <Exec Condition="$(OS) == 'Windows_NT'" Command="echo $(TargetDir) &gt; $(LocalAppData)\Logi\LogiPluginService\Plugins\$(ProjectName).link&#xD;&#xA;call $(ProjectDir)BuildEvents\postbuild.cmd $(ProjectDir) $(TargetDir)" />
+    <Exec Condition="$(OS) != 'Windows_NT'" Command="echo $(TargetDir) &gt; $(LocalAppData)\Logi\LogiPluginService\Plugins\$(ProjectName).link&#xD;&#xA;sh $(ProjectDir)BuildEvents\postbuild $(ProjectDir) $(TargetDir)" />
+  </Target>
+  <Target Name="PluginClean" AfterTargets="CoreClean">
+    <Delete Files="$(LocalAppData)\Loupedeck\Plugins\$(ProjectName).link" />

```

Comment:
The path should be `$(LocalAppData)\Logi\LogiPluginService\Plugins\$(ProjectName).link`.